pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go

```diff
@@ -0,0 +1,70 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope="Cluster"
+// +kubebuilder:storageversion
+
+// ResourceInterpreterCustomization describes the configuration of a specific
+// resource for Karmada to get the structure.
+// It has higher precedence than the default interpreter and the interpreter
```

Comment:
Can you explain it more clearly?

Why does `ResourceInterpreterCustomization` take precedence over `Resource Interpreter Webhook`?

Reply:
Sure, I can add more specific comment on it later.
For a quick answer, that is because the configurable interpreter would be more effective than webhook, that don't need to fire an HTTP request.